staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/time_test.go

```diff
@@ -171,3 +171,20 @@ func TestTimeProto(t *testing.T) {
 		}
 	}
 }
+
+func TestMarshalQueryParameterNilPointer(t *testing.T) {
+	var x *Time
+	v, err := x.MarshalQueryParameter()
+	if err != nil || v != "" {
+		t.Errorf("Unexpected results: '%s', %v", v, err)
+	}
+}
+
+func TestMarshalJSONNilPointer(t *testing.T) {
+	t.Skip("not sure if this should be supported")
```

Comment:
I'm not sure.  I'm somewhat concerned about this, but haven't had the time to think through implications.  I don't know that we have great coverage on bizarre time values.